### PR TITLE
ChatTranslate

### LINKED ISSUE #999: Chat Translator
Now I'm like ~70% sure that google translate has a `GET` API so all users would have to do would be provide an API key

### PR DESCRIPTION
**Pull request by sourTaste000**
**Describe the pull**
Added ChatTranslate Module (#999 )

**Describe how this pull is helpful**
Should close #999 when there is no error 

**Additional context**
Error:
```
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: java.lang.NoClassDefFoundError: com/google/cloud/translate/TranslateOptions
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at me.zeroeightsix.kami.module.modules.chat.ChatTranslate.<init>(ChatTranslate.java:27)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.lang.reflect.Constructor.newInstance(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at me.zeroeightsix.kami.module.ModuleManager.lambda$register$0(ModuleManager.java:38)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.SortedOps$SizedRefSortingSink.end(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.util.stream.ReferencePipeline.forEach(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at me.zeroeightsix.kami.module.ModuleManager.register(ModuleManager.java:36)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at me.zeroeightsix.kami.KamiMod.init(KamiMod.java:140)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:749)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:336)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:535)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraft.client.main.Main.main(SourceFile:123)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at java.lang.reflect.Method.invoke(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
[10:28:34] [Client thread/INFO] [STDERR]: [me.zeroeightsix.kami.module.ModuleManager:lambda$register$0:41]: 	at org.multimc.EntryPoint.main(EntryPoint.java:34)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: Caused by: java.lang.ClassNotFoundException: com.google.cloud.translate.TranslateOptions
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	... 64 more
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: Caused by: java.lang.NoClassDefFoundError: com/google/cloud/ServiceOptions
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.lang.ClassLoader.defineClass1(Native Method)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.lang.ClassLoader.defineClass(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.security.SecureClassLoader.defineClass(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	... 66 more
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: Caused by: java.lang.ClassNotFoundException: com.google.cloud.ServiceOptions
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[10:28:34] [Client thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:-1]: 	... 70 more
```

### CODE REVIEW

**Comment by sourTaste000** [comment]
It builds fine, but when i load the game it throws the error

**Comment by dominikaaaa** [comment]
you didn't include your deps in the shadowjar

File: build.gradle [hunk 1]
```diff
@@ -104,6 +104,7 @@ dependencies {
     compile group: "org.jetbrains", name: "annotations", version: annotations_version
     compile group: "org.jetbrains.kotlinx", name: "kotlinx-coroutines-core", version: coroutines_version
     compile group: "org.jetbrains.kotlinx", name: "kotlinx-coroutines-jdk8", version: coroutines_version
+    compile 'com.google.cloud:google-cloud-translate:1.95.0'
```
Comment (sourTaste000) [inline build.gradle:107]: here

File: build.gradle [hunk 2]
```diff
@@ -148,6 +149,7 @@ shadowJar {
         include(dependency("org.jetbrains.kotlinx:kotlinx-coroutines-core:${coroutines_version}"))
         include(dependency("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:${coroutines_version}"))
         include(dependency('net.shadowfacts:Forgelin'))
+        include(dependency('com.google.cloud:google-cloud-translate:1.95.0'))
```
Comment (sourTaste000) [inline build.gradle:152]: and here

File: build.gradle (hunk 1 above)
Comment (dominikaaaa) [reply build.gradle:107]: you need to also include it in the shadowjar

Comment (sourTaste000) [reply build.gradle:107]: https://github.com/kami-blue/client/pull/1009#discussion_r449668651

**Comment by sourTaste000** [comment]
ill  make another pull after i fix this problem lol

**Pull request closed without merging**